cypress/integration/channel/message_deletion_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 5] */
+
+describe('Message deletion', () => {
+    before(() => {
+        // 1. Go to Main Channel View with "user-1"
+        cy.toMainChannelView('user-1');
+    });
+
+    it('M13336 Delete both parent post and reply when deleting parent post from center', () => {
+        // 1. Post message in center.
+        cy.postMessage('test message deletion {enter}');
+
+        cy.getLastPostId().then((parentMessageId) => {
+            // 2. Mouseover the post and click post comment icon.
+            cy.clickPostCommentIcon();
+
+            // * Check that the RHS is open
+            cy.get('#rhsContainer').should('be.visible');
+
+            // 3. Post a reply in RHS.
+            cy.postMessageReplyInRHS('test message reply in RHS {enter}');
+
+            cy.getLastPostId().then((replyMessageId) => {
+                // 4. Click post dot menu in center.
+                cy.clickPostDotMenu(parentMessageId);
+
+                // 5. Click delete button.
+                cy.get(`#CENTER_button_${parentMessageId}`).click();
+                cy.get(`#delete_post_${parentMessageId}`).click({force: true});
```

Comment:
Test is failing on me, maybe due to extra click on center button.  Please remove ``cy.get(`#CENTER_button_${parentMessageId}`).click();``.  It can just be:

```suggestion
                cy.get(`#delete_post_${parentMessageId}`).click();
```